tests/ui/result_map_or_into_option.rs

```diff
@@ -1,8 +1,13 @@
-#![warn(clippy::result_map_or_into_option)]
+#![deny(clippy::result_map_or_into_option)]
 
 fn main() {
     let opt: Result<u32, &str> = Ok(1);
     let _ = opt.map_or(None, Some);
+    //~^ ERROR: called `map_or(None, Some)` on a `Result` value.
+    let _ = opt.map_or_else(|_| None, Some);
+    //~^ ERROR: called `map_or_else(|_| None, Some)` on a `Result` value
+    let _ = opt.map_or_else(|_| None, Some);
```

Comment:
Those test cases are exactly the same. Did you mean to add a `{ }` block around one of them, but `rustfmt` removed it?

Reply:
I didn't pay attention to rustfmt. Gonna need a skip here. ^^'